### PR TITLE
🐛 Fix multiple parameters of matcher (for prototype)

### PR DESCRIPTION
**Pull request by StewEucen**
## Why

* See #11

## How

* Fix to handle expected values in `createExpectCaller()`.

### CODE REVIEW

**Review by lisadhl — APPROVED** [review]
🍵

File: tests/__tests__/not/toBeCloseTo.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+require('../../../lib/setup').setup()
+
+describe('expect.each', () => {
+  test('#toBeCloseTo()', () => {
+    const actualValues = [
+      0.1 + 0.2,
+    ]
+    const expectedValues = [
+      0.3,
+    ]
```
Comment (StewEucen) [inline tests/__tests__/not/toBeCloseTo.js:13]: @lisadhl 

* I changed `~Table` to `~Values`. Please see last commt, and re-review.

**Review by StewEucen — COMMENTED** [review]
@lisadhl 

* Rename variables. Please re-approval.

File: tests/__tests__/not/toBeCloseTo.js (hunk 1 above)
Comment (lisadhl) [reply tests/__tests__/not/toBeCloseTo.js:13]: @StewEucen 

I'm sorry for not finding this.

Comment (StewEucen) [reply tests/__tests__/not/toBeCloseTo.js:13]: @lisadhl 

* This commit was made on the changes variable names, thus I have not noticed them.
錯誤是我們的朋友、没関係。

**Review by lisadhl — APPROVED** [review]

**Pull request merged**